nfsstat/README.md

```diff
@@ -0,0 +1,29 @@
+# Nfsstat Integration
+
+## Overview
+
+nfsiostat-sysstat is a tool that gets metrics from NFS mounts. This check grabs these metrics.
+
+## Installation
+
+Install the `dd-check-nfsstat` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `nfsstat.yaml` file to point to your server and port, set the masters to monitor
```

Comment:
We don't need to point to to the server or port, the only supported option is `nfsiostat_path`.